usaspending_api/common/retrieve_file_from_uri.py

```diff
@@ -45,6 +45,15 @@ def get_file_object(self):
         else:
             raise NotImplementedError("No handler for scheme: {}!".format(self.parsed_url_obj.scheme))
 
+    def copy(self, dest_file_path):
+        """
+            create a copy of the file and place at "dest_file_path"
+        """
+        if self.parsed_url_obj.scheme.startswith("http"):
```

Comment:
this still works for https, right?

Reply:
Yep. Tested locally with https and startswith should work for anything that begins with "http".